workers/ohsome_quality_analyst/api/api.py

```diff
@@ -188,32 +206,23 @@ async def _fetch_indicator(parameters) -> CustomJSONResponse:
     return CustomJSONResponse(content=response, media_type=MEDIA_TYPE_GEOJSON)
 
 
-@app.get("/report")
+@app.get("/report", tags=["report"])
 async def get_report(parameters=Depends(ReportDatabase)):
     """Request an already calculated Report for an AOI defined by OQT.
 
-    The response is a GeoJSON Feature with the Report results as properties.
     To request an Report for a custom AOI please use the POST method.
     """
     return await _fetch_report(parameters)
 
 
-@app.post("/report")
+@app.post("/report", tags=["report"])
 async def post_report(
     parameters: Union[ReportBpolys, ReportDatabase] = Body(
         ...,
         examples=REPORT_EXAMPLES,
     )
 ):
-    """Request a Report for an AOI defined by OQT or a custom AOI.
-
-    Either the parameters `dataset` and `featureId` have to be provided
-    or the parameter `bpolys` in form of a GeoJSON.
-
-    Depending on the input, the output is a GeoJSON Feature or FeatureCollection with
-    the indicator results. The Feature properties of the input GeoJSON will be preserved
-    if they do not collide with the properties set by OQT.
-    """
+    """Request a Report for an AOI defined by OQT or a custom AOI."""
```

Comment:
line 234 below might not be needed, as you do the same in line 239.

Reply:
Removed the duplicated code